app/selectors.js

```diff
@@ -703,6 +703,20 @@ export const filteredStakeTxs = createSelector(
   }
 );
 
+export const lastVotedTicket = createSelector(
+  [stakeTransactions],
+  (transactions) => {
+    const lastVotedTicket = Object.keys(transactions)
+      .map((hash) => transactions[hash])
+      .filter((transaction) => transaction.status=="voted")
```

Comment:
Use our constant from `constants/Decrediton.js`

Reply:
Fixed. Thanks for the suggestion!